kofta/src/app/pages/FollowListPage.tsx

```diff
@@ -29,6 +30,17 @@ export const FollowListPage: React.FC<FollowListPageProps> = () => {
 
 	const isFollowing = pathname.startsWith("/following");
 
+	useEffect(() => {
+    const fn = isFollowing ? setFollowingMap : setFollowerMap;
+    fn((m) => ({
+      ...m,
+      [userId]: {
+        ...m[userId],
+        loading: true,
+      },
+    }));
+	}, [isFollowing, setFollowerMap, setFollowingMap, userId]);
```

Comment:
the ws could return the data before useEffect is called and it would infinitely load, set loading to true when the wsend is called

Reply:
@benawad I'm a big fan, thank you so much for taking the time to review!

I used the same setFollowing/FollowerMap atom function inside the user profile, which is where the follow list is fetched from. Maybe this function could be abstracted into some other file, but I'm trying to keep the git diff small here.